deployments/liqo/values.yaml

```diff
@@ -292,7 +292,7 @@ telemetry:
   # -- Image repository for the telemetry pod.
   imageName: "ghcr.io/liqotech/telemetry"
   config:
-    # -- Set the schedule of the telemetry collector CronJob
+    # -- Set the schedule of the telemetry collector CronJob. Consider setting this value on ArgoCD deployments to avoid randomization.
```

Comment:
It is not random, but 10 minutes after you create the release. Since ArgoCD is continuously redoing helm templates, it is forever 10 minutes after now (if not set)

Reply:
Thought it was random since mine is set at 22 and I don't think I deployed it around that time 😬
Should I remove this comment or do you have any suggestion to improve it?

Reply:
made no further changes, please resolve if no action is required